assets/js/components/SocketHandler.jsx

```diff
@@ -12,64 +11,35 @@ class SocketHandler extends Component {
 
     this.subscribeToUpdates = this.subscribeToUpdates.bind(this)
     this.disconnect = this.disconnect.bind(this)
-    this.join = this.join.bind(this)
   }
 
-  componentDidMount() {
-    if (this.props.isLoggedIn && this.props.currentTeamId) {
-      this.props.fetchIndices()
-      this.subscribeToUpdates()
-    }
+  componentWillMount() {
```

Comment:
if we changed this to `<UserTeamProvider />`, what are we still using this component for?

Reply:
I envisioned this component as a wrapper for components that need socket access, gonna look into the graphql subscriptions tho